packages/core/src/modules/oob/messages/OutOfBandMessage.ts

```diff
@@ -0,0 +1,123 @@
+import type { PlaintextMessage } from '../../../types'
+import type { DidCommService } from '../../connections/models/did/service/DidCommService'
+
+import { Expose, Type } from 'class-transformer'
+import { ArrayNotEmpty, Equals, IsArray, IsInstance, IsOptional, ValidateNested } from 'class-validator'
+import { parseUrl } from 'query-string'
+
+import { AgentMessage } from '../../../agent/AgentMessage'
+import { Attachment, AttachmentData } from '../../../decorators/attachment/Attachment'
+import { AriesFrameworkError } from '../../../error'
+import { JsonEncoder } from '../../../utils/JsonEncoder'
+import { JsonTransformer } from '../../../utils/JsonTransformer'
+import { MessageValidator } from '../../../utils/MessageValidator'
+
+interface OutOfBandMessageOptions {
+  id?: string
+  label?: string
```

Comment:
Not optional to my understanding

Reply:
According to [OOB spec](https://github.com/hyperledger/aries-rfcs/tree/bed4989dd6517f7a9de3696800e57e4c6ef49231/features/0434-outofband#invitation-httpsdidcommorgout-of-bandverinvitation):  

> `label` - [optional] a self-attested string that the receiver may want to display to the user, likely about who sent the out-of-band message.

I think it makes sense. There is no reason for that to be required. It's not trusted attributed anyway. It's unnecessary because the receiver can decide what `label` the connection should have.

Reply:
Ah sorry my bad 👍